Open Judge System/Web/OJS.Web/Views/Account/ExternalNotify.cshtml

```diff
@@ -4,6 +4,11 @@
 
 <h2>Съжаляваме за неудобството</h2>
 <p>
-    Ако искате да се регистирате, променяте акаунта си или да възстановите своята парола, моля използвайте главния сайт на <a href="https://softuni.bg/account/authenticate" target="_blank">Софтуерния универсистет</a>.
+    Ако искате да се регистирате, променяте акаунта си или да възстановите своята парола, моля използвайте главния сайт на <a href="https://softuni.org/identity/externaljudgeregister" target="_blank">Софтуерния универсистет</a>.
```

Comment:
Can you use `Url.Action` here given that the link is external? Even if that's not possible you can still use the `Settings.PlatformBaseUrl` for the begging of the link.